packages/at_client/lib/src/service/sync_service_impl.dart

```diff
@@ -697,8 +699,8 @@ class SyncServiceImpl implements SyncService, AtSignChangeListener {
       try {
         command = await _getCommand(entry);
       } on KeyNotFoundException {
-        _logger.severe(
-            '${entry.atKey} is not found in keystore. Skipping to entry to sync');
+        _logger.info(
+            '${entry.atKey} is no longer in keystore. Skipping sync for it.');
```

Comment:
This is not an error, nor even a warning - just something that will happen from time to time in a very active keystore